### PR TITLE
Update new messages and their translations

### PR DESCRIPTION
**Pull request by 3d-gussner**
As with FW3.9.1 some new messages have been introduced here an updated translation version.

### CODE REVIEW

**Review by DRracer — APPROVED** [review]

File: lang/lang_en_fr.txt [hunk 1]
```diff
@@ -894,6 +902,10 @@
 "Please unload the filament first, then repeat this action."
 "SVP, dechargez le filament et reessayez."
 
+# c=20 r=4
+"Please check the IR sensor connection, unload filament if present."
+"SVP, verifiez la connexion du capteur IR et si le filament est decharge."
```
Comment (awenelo) [inline lang/lang_en_fr.txt]: I think that this should be `et decharge le filament`("and unload filament", instead of "check if the filament is unloaded")

Comment (DRracer) [reply lang/lang_en_fr.txt]: is it shorter than before ;) ?

Comment (DRracer) [reply lang/lang_en_fr.txt]: `SVP, verifiez la connexion du capteur IR et decharge le filament.`
this way?

Comment (awenelo) [reply lang/lang_en_fr.txt]: Yup!
65 vs 72 characters
```
New
SVP, verifiez la connexion du capteur IR et decharge le filament.
Old
SVP, verifiez la connexion du capteur IR et si le filament est decharge.
```

Comment (3d-gussner) [reply lang/lang_en_fr.txt]: Also used something similar in German translation as well as it makes it shorter and it makes sense
@awenelo Thanks again

Comment (awenelo) [reply lang/lang_en_fr.txt]: You're welcome!

**Pull request merged**